testplan/report/testing/base.py

```diff
@@ -327,9 +330,15 @@ class TestReport(BaseReportGroup):
     Only contains TestGroupReports as children.
     """
 
-    def __init__(self, meta=None, attachments=None, *args, **kwargs):
+    def __init__(self,
+                 meta=None,
+                 attachments=None,
+                 information=None,
+                 *args,
+                 **kwargs):
         self.meta = meta or {}
         self._tags_index = None
+        self._info = information
```

Comment:
Could you please add a comment to explain what is stored in info? Either here or on the property is fine.